core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java

```diff
@@ -86,6 +87,11 @@
   @Rule
   public ExpectedException mThrown = ExpectedException.none();
 
+  @Rule
```

Comment:
The test was using regular `@Before` so I used the corresponding `@Rule`. I think `@ClassRule` would work as well and be more consistent with other tests, so I'll switch it here, BlockMasterTest, and MasterSourceTest.